docs/hud_data_model.md

```diff
@@ -0,0 +1,100 @@
+# HMIS HUD CSV data model
+
+Based on:
+HMIS CSV FORMAT Specifications FY2024
+VERSION 1.4
+
+## Key Relationships
+
+* One Organization can have many Projects
+* One Project can have many Enrollments
+* One Client can have many Enrollments
+* Each Enrollment is tied to exactly one Project and one Client
+* Each Enrollment can have multiple associated services, assessments, and other records
+
+## ERD
+
+Note "CoC" is not a defined table; is included in the diagram as an "implied" entity
+
+```mermaid
+erDiagram
+    CoC ||..o{ Organization : "operates in"
+    CoC ||..o{ ProjectCoC : "contains"
+    Export ||--o{ Organization : contains
```

Comment:
I don't understand this one, it's not part of the HUD spec to my knowledge. ALL the HUD CSV files (tables) have an ExportID that references this record (not just  `Organization`).

Reply:
This was an error, thanks for catching